client/components/jetpack/what-is-jetpack/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
This component already exists on master (I copied it from this branch, I needed it :) thanks). So, rebasing this branch this file will disappear from the review

Reply:
Cool!